source/css/_objects.blocks.scss

```diff
@@ -106,7 +106,7 @@
 
 .c-block__button.o-button {
   position: relative;
-  display: flex;
+  display: inline-flex;
```

Comment:
Can we do `inline-flex` here? https://github.com/adventistchurch/alps/blob/f64189e2243a1f6c60ba9ef995697da28d919ae6/source/css/_objects.buttons.scss#L8

I'm still having issues with this:
![2019-12-05 at 10 20 AM](https://user-images.githubusercontent.com/132864/70248394-d6f9ca00-1748-11ea-9c18-3fc58b88149f.png)